regression/goto-harness/havoc-global-int-02/main.c

```diff
@@ -3,7 +3,7 @@
 unsigned int x;
 unsigned int y;
 
-unsigned int nondet_int()
+unsigned int some_random_unsigned_integer()
```

Comment:
This was changed because of a dump_c bug; It assumes that `nondet_*` names are available with specific meanings.